axelar/assetlist.json

```diff
@@ -1380,6 +1366,50 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ox.svg"
       }
+    },
+    {
+      "description": "The Axelar Bridged token of Cacao Swap",
+      "denom_units": [
+        {
+          "denom": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+          "exponent": 0
+        },
+        {
+          "denom": "yum.axl",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+      "name": "Axelar Wrapped YUM",
+      "display": "yum.axl",
+      "symbol": "YUM.axl",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
```

Comment:
this trace doesn't make sense. It cant be IBC transferred to the same chain it came from. Where does it originate?

Reply:
Hey @JeremyParish69 updated as originally from Arbitrum. Let me know if this is all good. Thanks (and sorry ive been travelling the past few days)

Reply:
The asset should then be registered in Arbitrum's assetlist and the image uploaded to /arbitrum/images/. Not sure why it's added to /ethereum/images/...